src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -268,11 +267,6 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 				let changedProp = changes[propName];
 				this._activeCellId = changedProp.currentValue;
 				this.toggleUserSelect(this.isActive());
-				// If the activeCellId is undefined (i.e. in an active cell update), don't unnecessarily set editMode to false;
-				// it will be set to true in a subsequent call to toggleEditMode()
-				if (changedProp.previousValue !== undefined) {
-					this.toggleEditMode(false);
```

Comment:
toggleEditMode() gets called when onCellEditModeChanged is fired - we don't need to call it again here.